### PR TITLE
Add k8s office hours

### PR DESCRIPTION
**Pull request by castrojo**
Adding a file for the time and specifics for the k8s office hours.

### CODE REVIEW

File: community/office-hours.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Kubernetes Community Office Hours
+
+Office Hours is a live stream where we answer live questions about Kubernetes from users on the [YouTube channel](https://www.youtube.com/c/KubernetesCommunity/).
+
+### When and Where
+
+Third Wednesday of every month, there are two sessions: 
+
+- European Edition: [1pm UTC](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Kubernetes+Office+Hours+%28EU+Edition%29&iso=20171018T13) 
+- Western Edition: [8pm UTC](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Kubernetes+Office+Hours+%28Western+Edition%29&iso=20171018T13&p1=234))
```
Comment (cblecker) [inline community/office-hours.md]: nit: extra ) at end

File: community/office-hours.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+
+### Volunteering
+
+We're always looking for volunteers to host and answer questions. Volunteers looking to host hours in 
+more time zones are also welcome to join in and help us expand. 
+
+- [Volunteer working sheet](http://bit.ly/k8s-office-hours-volunteers)
+
+If you have any questions ping @castrojo.
```
Comment (cblecker) [inline community/office-hours.md]: Github doesn't linkify markdown docs with usernames. You might want to do something like:
```
**[@castrojo](https://github.com/castrojo)**
```

**Comment by castrojo** [comment]
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/community/1144)

**Comment by castrojo** [comment]
@cblecker PTAL, thanks!

**Comment by cblecker** [comment]
/lgtm
:)

**Pull request merged**